src/Components/PokemonCard.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+export default function PokemonCard(props) {
+    const [pokemon, setPokemon] = useState(null);
+
+    useEffect(() => {
+
+        async function getPokemonData() {
+            const result = await axios.get(
+                `https://pokeapi.co/api/v2/pokemon/${props.name}`
+            );
+
+            setPokemon(result.data);
+        }
+
+        getPokemonData();
+    }, [props.name]);
+
+    return (
+        <div className="pokemonCard">
+            <h1>{pokemon?.name}</h1>
```

Comment:
De `?` operator zorgt ervoor dat als de `name` property niet bestaat op het pokémon object, er geen error komt. Het checkt alleen niet of het pokemon object uberhaupt wel data bevat, dus die `{pokemon && check` hebben we nog wel nodig!